src/objects/entities/Entity.ts

```diff
@@ -11,35 +10,30 @@ type AnimationsConfigType = {
   repeat?: number | undefined;
 };
 
-type EntityConfigType = {
+export type EntityConfigType = {
   name: string;
   spriteSheetKey: string;
   animations: AnimationsConfigType[];
-  physicsConfig?: Phaser.Types.Physics.Matter.MatterSetBodyConfig;
-  keepUprightStratergy: KeepUprightStratergies;
+  physicsConfig?: MatterJS.IChamferableBodyDefinition & {
```

Comment:
I tried using `IChamferableBodyDefinition ` before but it doesn't have a type param. We were passing this to create the hitbox and type was used to tell it which shape for the collision, circle|rectangle and some others. 

Actually I'm not sure how that was working as `physicsConfig `is past to `Bodies.rectangle()` 🤔. It's fine for now but If we want diff collision shapes we'll have to do a check on `physicsConfig.type` then pass it to either `Bodies.circle()` or `Bodies.rectangle()`. There's a few other shapes as well.